modules/gdnative/godot/godot_string.cpp

```diff
@@ -112,6 +112,188 @@ void GDAPI godot_string_destroy(godot_string *p_self) {
 	self->~String();
 }
 
+/* Helpers */
+
+godot_bool GDAPI godot_string_begins_with(const godot_string *self, const godot_string *string) {
```

Comment:
Usually all the parameters are prefixed with `p_`, like `const godot_string *p_self`. You can see that that usually is the case through all the engine 😄